### ISSUE TITLE
Is this module maintained?

### ISSUE THREAD

**Issue by clarkie** [ryanfitz/vogels#171]
I know this is very subjective but I'm concerned that @ryanfitz hasn't made any movement on some of the PRs & issues for 4 months (ish). Some options:
1. Fork, patch and move on
2. Add some more maintainers to this repo + npm

I'm happy to volunteer to help although my experience using this module is about 3-4 weeks and very limited. I wondered if there would be any other offers of help?

This isn't a dig, whine, etc that is common place in the JS community these days but an observation and hopefully a start to give @ryanfitz some help looking after this. I'd like to see vogels move forward as it's by far the best abstraction over dynamodb I have come across.

**Comment by justinffs** [comment]
I would be willing to volunteer as well

**Comment by sylwit** [comment]
volunteer also

**Comment by clarkie** [comment]
Cool, that's 3 of us. How do we do this then? @ryanfitz - any suggestions?

**Comment by tony-gutierrez** [comment]
It looks like I am going to be using this library. Id also volunteer to work on it. @ryanfitz can we get your input on a strategy for moving forward?

**Comment by shaunbruno** [comment]
I am interested in using this lib as well - I'll hang tight for a bit for @ryanfitz to offer feedback ^

**Comment by tony-gutierrez** [comment]
This looks farther along? 
https://github.com/automategreen/dynamoose

**Comment by justinffs** [comment]
@tony-gutierrez  Seems to be missing quite a bit
No table.update https://github.com/automategreen/dynamoose/blob/master/lib/Table.js#L327
And it looks like there is only ability to add indexes when the table is created.

Also using Q which is Ok but using bluebird js would be better.

The documentation and examples also seem to be a bit less than what vogels has.

**Comment by tony-gutierrez** [comment]
True enough. Don't want to hijack this thread.

**Comment by allenhartwig** [comment]
I'm interested in seeing this project live on as well. Being that we are going on a month with no response from @ryanfitz on this issue should we start taking the fork approach?

**Comment by ruddiculous** [comment]
I haven't had any response since I updated https://github.com/ryanfitz/vogels/pull/141 back in Feb, FYI.

**Comment by jkeczan** [comment]
I would be willing to help as well as my company would like to use this as well....

**Comment by codedhrj** [comment]
We are planning to use this module, but as we all can see, this is not maintained.
I would like to know what other people are doing? Are you still using this module despite the fact it is not maintained anymore or you started using any other module?

**Comment by clarkie** [comment]
Right, I've probably set myself up for a whole world of pain but I've forked but not in a fork:

https://github.com/clarkie/dynogels
https://www.npmjs.com/package/dynogels

I'm going to try and be as active over there as I possibly can but will definitely need some help so please chip in. 

All credit will obviously still go to @ryanfitz for his efforts on getting vogels off the ground.

Thanks

**Comment by AaronHarris** [comment]
I sent a [tweet](https://twitter.com/HarrisAaron/status/762820066199031808) to @ryanfitz, hopefully we will hear a response.

**Comment by marckaraujo** [comment]
Any news? This library seems to be the best one, dont let it die.

**Comment by ngocketit** [comment]
Still no reply from @ryanfitz ? Maybe he's retired already. We should support @clarkie for his clone and move it forward.

**Comment by marckaraujo** [comment]
Yes, @ngocketit . You should use https://github.com/clarkie/dynogels. Its very nice and up to date.

**Comment by servel333** [comment]
@clarkie This might be a good opportunity to create an organization and transfer the project to that in order to help better maintain it going forward.
